src/EditorFeatures/Core/Implementation/Organizing/OrganizeDocumentCommandHandler.cs

```diff
@@ -40,7 +40,17 @@ public bool ExecuteCommand(OrganizeDocumentCommandArgs args, CommandExecutionCon
         {
             using (context.OperationContext.AddScope(allowCancellation: true, EditorFeaturesResources.Organizing_document))
             {
-                this.Organize(args.SubjectBuffer, context.OperationContext);
+                var cancellationToken = context.OperationContext.UserCancellationToken;
+                var document = args.SubjectBuffer.CurrentSnapshot.GetFullyLoadedOpenDocumentInCurrentContextWithChangesAsync(context.OperationContext)
+                    .WaitAndGetResult(cancellationToken);
```

Comment:
this seems odd.  Organizing is syntactic.  Waiting on a fully loaded solution seems unnecessary.  This also seems to be mixing in an unneeded change into thsi PR.

Reply:
This was already being waited on, I just moved it.  

As far as I remember, organize was using semantic information - http://source.roslyn.io/#Microsoft.CodeAnalysis.Features/Organizing/Organizers/ISyntaxOrganizer.cs,8e517d2bd0cd7671

Reply:
1. gotcha on the move.
2.  That's just wacky (and likely due to how ancient this code is).  No actual impl ever uses the SemanticModel.  You can see that in the only impl here:

```c#
        public SyntaxNode OrganizeNode(SemanticModel semanticModel, SyntaxNode node, CancellationToken cancellationToken)
        {
            return Organize((TSyntaxNode)node, cancellationToken);
        }
```

The semantic model is just dropped, and only the node is used.  Must be a holdover from when we though we might use it.

Reply:
@heejaechang should we change this one for partial load? 

If we don't wait on this one, I would prefer to remove the semantic model all together from the interface along with removing the wait check.  I'd do that in a separate PR though.

Reply:
I'm personally fine with this being in a followup PR.

Reply:
ya. I saw the semantic model stuff as well. and saw it wasn't used in our implementation. but it wasn't releated to partial load, so didn't touch it. fine removing it if nothing goes bad.